app/src/main/java/com/android/streetworkapp/ui/profile/Profile.kt

```diff
@@ -167,14 +177,16 @@ fun DisplayFriendList(friends: List<User?>) {
 }
 
 /**
- * This function displays a friend (for the friend list).
+ * This function displays a friend (for the friend list). Also displays actions to perform on them
+ * through a dropdown menu.
```

Comment:
To make the documentation easier to read, I'd write it in a single sentence:

 “_This function displays a friend (for the friend list) and actions to perform on them through a dropdown menu._”.

It's only a suggestion, I leave it to you to choose what you prefer.

Reply:
I agree that this could be improved. I changed it to :

"_This function displays a friend (of the friend list) and actions about them through a dropdown menu._"